utils/testsuite/testsuite_skiplist.py

```diff
@@ -1900,6 +1900,8 @@
     "cross-realm",
 ]
 
+INTL_SKIP_LIST = []
```

Comment:
Let's actually rename this to something more descriptive. Perhaps something like `SUPPORTED_INTL_TESTS`. Calling it a skip list can cause confusion as to why the Intl tests are split across two different skip lists.